### PR TITLE
HDH-201 migrate-videos-from-iframe-to-doc-video-component

### PR DESCRIPTION
**Pull request by rohanmaharjan100**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by aimurphy — APPROVED** [review]

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/1-auto-stopping-rules.md [hunk 1]
```diff
@@ -15,7 +15,8 @@ AutoStopping Rules make sure that your non-production resources run only when us
 * Stop paying for cloud waste
 
 ### Visual Summary
-<figure><iframe src="//fast.wistia.com/embed/iframe/8wo6shjqqh" width="560" height="315" frameborder="0" allowfullscreen=""></iframe></figure>
+
+<docvideo src="//fast.wistia.com/embed/iframe/8wo6shjqqh" />
```
Comment (aimurphy) [inline docs/cloud-cost-management/4-use-ccm-cost-optimization/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/1-auto-stopping-rules.md:19]: This is still a wistia link

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/4-create-auto-stopping-rules/autostopping-dashboard.md [hunk 2]
```diff
@@ -26,9 +26,7 @@ You can also perform the following actions from the AutoStopping dashboard view:
 
 ![](./static/autostopping-dashboard-31.png)
 
-
-### Visual summary
-<figure><iframe src="//fast.wistia.com/embed/iframe/b7i08zy2yt" width="560" height="315" frameborder="0" allowfullscreen=""></iframe></figure>
+<docvideo src="//fast.wistia.com/embed/iframe/b7i08zy2yt" />
```
Comment (aimurphy) [inline docs/cloud-cost-management/4-use-ccm-cost-optimization/1-optimize-cloud-costs-with-intelligent-cloud-auto-stopping-rules/4-create-auto-stopping-rules/autostopping-dashboard.md:29]: this is still wistia

**Comment by aimurphy** [comment]
Approved wrt CI topics, but I still see Wistia links in other areas.

**Comment by ravilach** [comment]
@rohanmaharjan100 I think we can close this, we took forward in other PRs. Thanks!

**Comment by rohanmaharjan100** [comment]
Yeah..lets close it.

**Pull request closed without merging**